lib/tasks/form_definition.rake

```diff
@@ -0,0 +1,6 @@
+namespace :form_definition do
+    desc "Validate JSON against Generic Form Schema"
+    task :validate do
+        ruby "app/helpers/json_schema_validation.rb" 
```

Comment:
Ah sorry I think this wasn't very clear. I'd move that whole files logic into here rather than keeping a file in the app dir for this.

Reply:
I moved it into lib along with the rake file but think its cleaner as seperate file?

Reply:
Ok all in one now